official/benchmark/benchmark_uploader_main.py

```diff
@@ -54,6 +54,10 @@ def main(_):
   uploader.upload_metric_file(
       flags.FLAGS.bigquery_data_set, flags.FLAGS.bigquery_metric_table, run_id,
       metric_json_file)
+  # Assume the run finished successfully before user invoke the upload script.
+  uploader.insert_run_status(
+      flags.FLAGS.bigquery_data_set, flags.FLAGS.bigquery_run_status_table,
+      run_id, "success")
 
```

Comment:
In the event of run failure, the status should be updated. Can we `try, catch: update_run_status(failure), raise`? In the case of catastrophic failure (ie, machine down or sigterm), the run will be left in limbo, but I don't think we need to handle that currently, since the status is for tracking rather than transaction locking or anything particularly sensitive. We can always clear after the fact.